pyclean/modern.py

```diff
@@ -88,9 +88,25 @@ def descend_and_clean_bytecode(directory):
             if child.suffix in ['.pyc', '.pyo']:
                 Runner.unlink(child)
         elif child.is_dir():
-            descend_and_clean_bytecode(child)
+            if child.name == '.pytest_cache' and clean_test_cache:
+                descend_and_remove_directory(child)
+            else:
+                descend_and_clean_bytecode(child, clean_test_cache)
 
             if child.name == '__pycache__':
                 Runner.rmdir(child)
```

Comment:
Here we're already removing a folder. In theory you could simply add `'.pytest_cache'` here as a candidate to remove, couldn't you?